components/financial_assistance/app/views/financial_assistance/incomes/_job_income_form.html.erb

```diff
@@ -1,78 +1,145 @@
-<%= income_form_for @application, @applicant, income do |f| %>
-  <% income_id = income.id %>
-  <% onchange_date = FinancialAssistanceRegistry[:income_and_deduction_date_warning].enabled? ? (raw "validateDateWarnings('#{income_id}')") : "" %>
-  <%= f.hidden_field :kind, value: 'wages_and_salaries' %>
-  <div class="focus_effect_module row-edit-border">
-    <div class="row row-form-wrapper no-buffer row-height">
-      <div class="col-md-12">
-        <%= f.text_field :employer_name, placeholder: 'EMPLOYER NAME *', required: true, class: "floatlabel" %>
+<% if @bs4 %>
+  <%= income_form_for @application, @applicant, income do |f| %>
+    <%= hidden_field_tag :bs4, true %>
+    <% income_id = income.id %>
+    <% onchange_date = FinancialAssistanceRegistry[:income_and_deduction_date_warning].enabled? ? (raw "validateDateWarnings('#{income_id}', true)") : "" %>
+    <%= f.hidden_field :kind, value: 'wages_and_salaries' %>
+    <div class="alert alert-info d-flex" role="alert">
+      <div class="pl-2 pt-2 pr-1">
+        <div class="info-icon icon" alt="info" aria-hidden="true">&nbsp;</div>
+      </div>
+      <div class="mt-1 pl-0 pr-2 pt-1">
+        <p class="subtext"><%= l10n('faa.incomes.job_end_date_note', subject: @applicant.first_name, income_type: "job") %></p>
```

Comment:
Can we reuse the `faa.incomes.income_note` for this content? If so, we can get rid of this key in the translations file as well